src/tests/event.controller.test.ts

```diff
@@ -0,0 +1,25 @@
+import { EventService } from '@Services/event.service';
+import { AppUserService } from '@Services/app-user.service';
+import { MockEventService } from './mocks/event.service.mock';
+import { MockAppUserService } from './mocks/app-user.service.mock';
+
+import { EventController } from '@Controllers/event.controller';
+
+import { Container } from 'typedi';
+
+describe('EventController', () => {
+  beforeEach(() => {
+    const mockEventService = new MockEventService(new Map());
+    const mockAppUserService = new MockAppUserService();
+    Container.set(EventService, mockEventService);
+    Container.set(AppUserService, mockAppUserService);
+  });
+
+  describe('createEvent', () => {
```

Comment:
What does this test currently do exactly? As far as I can tell it's only checking if true is equal to true.

Reply:
yeah - it only check true is equal to true. it was mainly a POC to demonstrate how to mock stuff with typedi